Anton_Bartoshevich/08.Docker.Docker-compose/docker-image.yml

```diff
@@ -0,0 +1,18 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag my-image-name:$(date +%s)
```

Comment:
this is only build, there is no push in your github action

Reply:
Sorry, the .yml file is wrong, I uploaded the correct .yml file.